maintainers/maintainer-list.nix

```diff
@@ -6149,10 +6149,10 @@
     githubId = 6455574;
     name = "Matt Votava";
   };
-  mwilsoninsight = {
-    email = "max.wilson@insight.com";
-    github = "mwilsoninsight";
```

Comment:
Can you approve this PR with the old account to verify?

Reply:
I would love to, but I don't have access to the email address that backs it anymore (stupid past-me didn't think to close it out after I last used it in May 2019). :sweat: 

I just opened a ticket with GitHub Support to see how I can recover it (if at all- the machine I used (ssh key) has been returned to my former employer as well).

Other than account activity, I don't really have another form of verification other than I really really promise it's me. lol

Reply:
AHA!

I found the discourse discussions I was having juuuuust prior to leaving that account behind.

Check out [this comment](https://discourse.nixos.org/t/nixops-travis-ci-test-failure-on-azure-private-ip-pr/2441/10?u=mwilsoncoding) from April '19 where I posted a link to my PR (which I submitted as `mwilsoninsight`) while logged in under my more persistent account: `mwilsoncoding`

^ That's about the strongest form of proof I have at the moment.

Reply:
I do remember, but without some verification from the old account, there's still the possibility those two accounts belonged to two separate people.

I don't think there's a whole lot to gain by impersonating someone else in NixOS/Nixpkgs; in our current state, all this really translates to is getting spammed with github notifications.

I think it would be okay if @mwilsoninsight doesn't respond, that we can assume that the previous user no longer exists.

Reply:
As a follow up, I was unable to regain access to the account without access to the email associated with it.

GitHub support was nice enough to hide the account tho, so it should appear as deleted.